test/integration/puppetlabs/services/master/master_service_test.clj

```diff
@@ -766,3 +766,26 @@
 
             (is (= 503 status-code))
             (is (<= 0 retry-after 1800))))))))
+
+(deftest ^:integration project-file-content
```

Comment:
Was there a plan to do any testing with the `named` project?
There seem to be a handful of unused test files (though maybe some are just necessary for structuring a project correctly/realistically)?

Reply:
Originally I had a question of whether we ever needed to identify a project by it's name from the yaml file when that's different from the directory name. I think that we should just avoid this though, since we're going to write how deployment works.

Reply:
hmm, I think it makes sense to avoid that here. but I'm wondering where in the workflow that needs to happen.. Do we allow a user to deploy a project using the name from the yaml file (as opposed the name of the directory/repo)? If so, will the front end handle that or do we need to handle that in code-manager? 

If we're not gonna deal with it here, I think we should remove the associated test files.